entries/jQuery.when.xml

```diff
@@ -12,10 +12,15 @@
     <p>If no arguments are passed to <code>jQuery.when()</code>, it will return a resolved Promise.</p>
     <p>If a single Deferred is passed to <code>jQuery.when()</code>, its Promise object (a subset of the Deferred methods) is returned by the method. Additional methods of the Promise object can be called to attach callbacks, such as <a href="/deferred.then/"><code>deferred.then</code></a>. When the Deferred is resolved or rejected, usually by the code that created the Deferred originally, the appropriate callbacks will be called. For example, the jqXHR object returned by <code>jQuery.ajax()</code> is a Promise-compatible object and can be used this way:</p>
     <pre><code>
-$.when( $.ajax( "test.aspx" ) ).then(function( data, textStatus, jqXHR ) {
-  alert( jqXHR.status ); // Alerts 200
+$.when( $.ajax( "test.aspx" ), $.ajax( "page.aspx" ) ).then(function( res1, res2 ) {
+  alert( res1[2].status ); // Alerts 200
+})
+.fail(function( err1, err2 ) {
```

Comment:
Should we switch to `.catch()` here @gibson042? Also since the failure occurs as soon as any rejection occurs, I think it's possible that `res1` does not yet have a status if `res2` rejected. If a programming exception occurred they might even be `Error` objects instead of `jQXHR` objects! In real code you'd need to inspect both to sort it out.

Perhaps for the purposes of this example we could just `alert("one or more requests failed")`.